markdown/dev/reference/api/part/props/en.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Part.asProps()
+---
+
+This `Part.asProps()` method can be used to return the part as an object that is suitable for inclusion in renderprops. 
+
+
+## Example
+
+<Example caption=" Example of the Part.asprops() method">
```

Comment:
These examples are specific. They require a draft method. If you want to include some other example, just wrap it in a js codeblock:

````
```js
const myExample = 'foo'
```
````